src/shared-components/optionButton/index.tsx

```diff
@@ -40,41 +41,46 @@ export const OptionButton = ({
   subtext,
   text,
   ...rest
-}: OptionButtonProps) => (
-  <ClickableContainer
-    borderRadius={borderRadius}
-    onClick={onClick}
-    type="button"
-    role={optionType}
-    containerType="clickable"
-    // eslint-disable-next-line react/jsx-props-no-spreading
-    {...rest}
-  >
-    <FlexContainer>
-      {icon ? (
-        <IconWrapper
-          selected={selected}
-          optionType={optionType}
-          buttonType={buttonType}
-        >
-          {selected ? <CheckmarkIcon /> : icon}
-        </IconWrapper>
-      ) : (
-        <CheckmarkWrapper
-          selected={selected}
-          optionType={optionType}
-          buttonType={buttonType}
-        >
-          <CheckmarkIcon />
-        </CheckmarkWrapper>
-      )}
-      <TextContainer>
-        <Text>{text}</Text>
-        {subtext && <SubText>{subtext}</SubText>}
-      </TextContainer>
-    </FlexContainer>
-  </ClickableContainer>
-);
+}: OptionButtonProps) => {
+  const theme = useTheme();
+  const borderRadiusValue = borderRadius || theme.BORDER_RADIUS.small;
```

Comment:
Let's kick this to the styled component logic--it'll keep the diff smaller, too.

Reply:
done